README.md

```diff
@@ -63,7 +65,38 @@ The valid project types are:
 - `classlib` for Class Library
 - `unittest` Unit Test project
 
-> Example: `yo aspnet webbasic "my test app"` will create a "Web Application Basic" project called "my test app"
+The valid UI framework types are:
+
+- `bootstrap` for Bootstrap (this is the default and does not have to be specified explicitly)
+- `semantic` for Semantic UI
+
+> Example: `yo aspnet webbasic "my semantic app" semantic` will create a "Web Application Basic" project called "my semantic app" using the Semantic UI framework.
+
+> Example: `yo aspnet webbasic "my bootstrap app"` OR `yo aspnet webbasic "my bootstrap app" bootstrap` will create a "Web Application Basic" project called "my bootstrap app" using the Bootstrap framework.
+
+## Additional UI Framework Notes
+
```

Comment:
@peterblazejewicz Is there anything I can do here or was this more directed towards @sayedihashimi ? I have no access to create Wiki pages, etc...

If you would like I could amend the PR notes with more details explaining the conversation we had, why the PR was requested, etc...